packages/@okta/vuepress-theme-prose/components/SidebarItem.vue

```diff
@@ -10,7 +10,7 @@
             </svg>
 
             <div v-if="link.path">
-                <router-link :to="link.path" @click="setData" :class="{'router-link-active': link.imActive, 'router-link-exact-active': link.imActive}">{{link.title}}</router-link>
+                <router-link :to="link.path" @click="setData" :class="{'tree-nav-link': true, 'router-link-active': link.imActive, 'router-link-exact-active': link.imActive}">{{link.title}}</router-link>
```

Comment:
Do we need both "router-link-active" and "router-link-exact-active"? I mean, both of classes are dependent on link.imActive property

Reply:
it looks like these classes were used to toggle font weight under [`.overview`](https://github.com/okta/okta-developer-docs/blob/31f38a4ee918658663ddcce18913891f4231b1f4/packages/%40okta/vuepress-theme-prose/assets/css/components/_sidebar.scss#L50) section (though markup with `.overview` class doesn't seem to be around anymore)
I agree we can go with one class now, however I suggest we cover it in OKTA-320822(i.e. remove bold font-weight for non-ancestors by dropping `.router-link-active`).

Reply:
Got it, thanks. Will merge this changes to me.